cypress.config.js

```diff
@@ -1,9 +1,26 @@
 const { defineConfig } = require('cypress');
+const faker = require('faker');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com',
```

Comment:
The baseUrl is hardcoded. Consider using environment variables to set the baseUrl, making your tests more flexible to different environments.